tensorflow/contrib/rnn/kernels/BUILD

```diff
@@ -0,0 +1,19 @@
+# Libraries and Kernels for RNNs.
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+package(default_visibility = ["//tensorflow:__subpackages__"])
+
+load("//tensorflow:tensorflow.bzl", "tf_kernel_library")
+
+tf_kernel_library(
+    name = "lstm_ops",
+    prefix = "lstm",
+    deps = [
+        "//tensorflow/contrib/rnn:lstm_ops_op_lib",
+        "//tensorflow/core:framework",
```

Comment:
That's correct. That's why I have `tf_custom_op_library` handle it specially. Can you please follow the recipe here https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/how_tos/adding_an_op/BUILD#L100 for your `.so`?